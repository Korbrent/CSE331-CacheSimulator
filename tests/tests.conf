s 00000401 1
s 00000402 1
s 00000403 1
s 00000404 1
l 00000405 1
l 00000401 1
l 00000801 1
s 00000405 1
s 00000802 1
l 00000402 1
l 00000805 1
s 00000405 1
l 00000405 1 