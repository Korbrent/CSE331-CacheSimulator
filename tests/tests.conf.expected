s 00000401 1      M      M
s 00000402 1      M      M
s 00000403 1      M      M
s 00000404 1      M      M
l 00000405 1      M      M
l 00000401 1      H      M
l 00000801 1      M      M
s 00000405 1      H      H
s 00000802 1      M      M
l 00000402 1      M      H
l 00000805 1      M      M
s 00000405 1      M      M
l 00000405 1      H      M